_posts/2020-01-26-celebrate_1k_stars.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Celebrating the first 1000 stars on GitHub"
+date:   2020-01-26
+categories: [jump-dev]
+permalink: /blog/1kstars/
+---
+
+JuMP just received its 1000th star on GitHub! We will use this milestone to recap JuMP's growth and other important milestones since its beginnings as an experimental project in 2012.
+
+Tracking the success of some open-source projects can be hard because we might have access to the number of downloads (JuMP's case due to Julia's packge manager implementation), but we can check some metrics like the number of contributors, citations to code or papers describing the project.
+The number of citations of the two main JuMP papers are 253+428 (with some possible double counting).
+Currently 71 developers have added some contribution in some of the 3747 commits of the JuMP repository made available in 67 releases. And this is only data from the main JuMP repo. The ecosystem has many other repos ranging from MOI to solver wrappers.
+
+The number of stars of a GitHub project is yet another indicator, because it means people are adding the project to their list of favorites. However, people that starred JuMP are no necessarily users and many users don't star JuMP (if thats you case give us your star!), therefore its not a precise metric.
+
+Reaching these 1k stars is a very happy moment for the developers that can see some feedback on their work and motivates more work and new developments.
+
+
+<img src="/assets/news/1kstars/1kstars_plot.jpg" alt="The growth of JuMP 2012 - 2020. Obtained with https://star-history.t9t.io/">
+
+
+This will be added to the greatest moments timeline:
+
+* 09 Oct 2012 - First commit to JuMP.jl repo
+* 15 Dec 2013 - JuMP 0.2.0 - Solver Callbacks (XXX stars)
+* 02 May 2014 - JuMP 0.5.0 - Nonlinear optimization (XXX stars)
+* 10 Dec 2014 - First Hundred stars!
+* XX YYY 2015 - First JuMP presentation in JuliaCon 2015 (XXX stars)
+* 16 Mar 2015 - First JuMP paper published: Computing in Operations Research Using Julia (117 stars)
+* 31 Aug 2015 - JuMP 0.10.0 - Semidefinite optimization (XXX stars)
+* 15 Dec 2016 - INFORMS Computing Society (ICS) Prize (290 stars)
+* 05 May 2017 - Second JuMP paper published: JuMP: A Modeling Language for Mathematical Optimization (330 stars)
+* 12 Jun 2017 - First JuMP developers meeting in Cambridge, USA (XX participants) (339 stars)
```

Comment:
I would call it "JuMP-dev workshop" instead of "JuMP developers meeting" for consistency.